source/server.js

```diff
@@ -305,11 +305,11 @@ app.get('/api/gitversion', (req, res) => {
 const userConfigPath = path.join(config.homedir, '.ungitrc');
 const readUserConfig = () => {
   return fs.isExists(userConfigPath).then((hasConfig) => {
-      if (!hasConfig) return {};
-      return fs.readFileAsync(userConfigPath, { encoding: 'utf8' }).then((content) => {
-          return JSON.parse(content.toString());
-        });
-    });
+    if (!hasConfig) return {};
+    return fs.readFileAsync(userConfigPath, { encoding: 'utf8' })
+      .then((content) => { return JSON.parse(content.toString()); })
+      .catch((err) => { winston.error(`Ignore .ungitrc because ${err}`); return {}; });
```

Comment:
If `.ungitrc` is malformed or errored, should we throw exception here to halt the entire system rather than starting the system?